bridge-fungible-token/src/data.sw

```diff
@@ -1,10 +1,10 @@
-library data;
-
-use std::{address::Address, contract_id::ContractId};
+library;
 
 pub struct MessageData {
     token: b256,
     from: b256,
-    to: Address,
+    to: Identity,
     amount: b256,
+    deposit_to_contract: bool,
```

Comment:
It seems like this variable is never read from? Do we really need this if we already have the `Identity` enum which could be checked to see if this is a contract deposit?

Reply:
You're right, a recent refactor means this field is no longer needed.
The length of the message (cached in the `len` field) is sufficient for identifying the inted=nded recipient type.
Until we check the length of the message, all we have is a `b256` value for the recipient; we assign the Identity type after checking the msg length.